src/Microsoft.ML.FastTree/FastTreeArguments.cs

```diff
@@ -20,6 +20,7 @@ public interface IFastTreeTrainerFactory : IComponentFactory<ITrainer>
     {
     }
 
+    /// <include file='./XMLDoc.xml' path='docs/members/member[@name="FastTree"]/*' />
```

Comment:
>XMLDoc [](start = 25, length = 6)

I might be being a little OCD, but the redundancy of the name `XMLDoc.xml` just drives me absolutely bonkers. Maybe just plain old `doc.xml` (lower case `doc`)?